README.md

```diff
@@ -4,6 +4,107 @@ README isn't up-to-date.
 
 // TODO(user): Add simple overview of use/purpose
 
+## Building the Project
+
+This project uses [just](https://github.com/casey/just) as a command
+runner (similar to `make`). Follow these steps to build the project from
+source.
+
+### Prerequisites
+
+- **Go** 1.21 or later
+- **just** command runner - Install with:
+  ```sh
+  # On macOS
+  brew install just
+
+  # On Linux
+  cargo install just
+  # Or download from https://github.com/casey/just/releases
+  ```
+### Build Steps
+
+1. **Install build tools**
+
+   The project requires several Go-based build tools. Install them all with:
+   ```sh
+   just tools
+   ```
+
+   This installs the following tools into the `bin/` directory:
+   - `kustomize` - Kubernetes manifest customization tool
+   - `controller-gen` - Kubernetes code and manifest generator
+   - `golangci-lint` - Go linter
+   - `helm` - Kubernetes package manager
+   - `crd-ref-docs` - CRD documentation generator
+   - And other supporting tools
+
+   **Note:** If the tools are installed in `hack/bin/` instead of `bin/`, move them:
```

Comment:
Hm, that's very strange, we're not seeing it on both dev machines and in CI. What version of just do you have? AFAIR at least 1.36 is required, so we should rather document it as a requirement other then providing the workaround.